### ISSUE TITLE
Support for application name argument 

### ISSUE THREAD

**Issue by isaiasmatewos** [doubledotlabs/doki#7]
Hey There,

Thank your for making doki! 

Can you add support for app name url paramater:

e.g https://dontkillmyapp.com/xiaomi?app=AppNameHere 

This allows you to use your apps real name in the guides and references.

Thanks

**Comment by fennifith** [comment]
Hmm, it looks like dontkillmyapp just finds all occurrences of the text `"your app"` on the page and replaces them with whatever you pass to the URL parameter. This should definitely be possible, if a bit strange to implement (lol)

**Comment by isaiasmatewos** [comment]
Yeah it's definitely possible, I could help with implement it if you like
